### DISCUSSION TITLE
polymath default to same collection as track

### DISCUSSION THREAD

**Discussion by jhurlbut** [okio-ai/nendo-platform#37] [Ideas]
It would be nice if polymath would default add to the same collection as the original track but I guess that would be problematic if the track was in multiple collections.

If you run mashuper on a collection will it automatically pull from the related stem loops of tracks in the selected collection if those loops don't belong to the collection?

**Comment by faradox** [comment]
We thought it might be better to leave it up to the user to decide whether they want polymath's results to be added to the source collection. There is a collection selector in the tool settings that allows to configure exactly what you want.

The answer to your question is no. The mashuper currently searches for loops directly in the collection you are running it on. It will only use related quantized loops if it finds any that already have the desired target bpm. Your idea is good though, will consider changing the mashuper accordingly.
